dev-qt/qtwebengine/qtwebengine-5.15.0.ebuild

```diff
@@ -9,9 +9,13 @@ inherit multiprocessing python-any-r1 qt5-build
 DESCRIPTION="Library for rendering dynamic web content in Qt5 C++ and QML applications"
 
 if [[ ${QT5_BUILD_TYPE} == release ]]; then
-	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
 fi
 
+# patchset unlikely to change on minor bumps, hardcode version
+# ppc64 patchset maintained by gyakovlev@gentoo.org
```

Comment:
do we have a way to express this in `metadata.xml`?

Reply:
sure, I can add myself there.